cmd/server-manager/typescript/src/RaceControl.ts

```diff
@@ -116,6 +116,49 @@ export class RaceControl {
             case EventNewSession:
                 this.showTrackWeatherImage();
                 break;
+            case EventChat:
+                let $chatContainer = $("#chat-container");
+
+                let chatMessage = $("#chat-message-template").first().clone();
```

Comment:
can this use a class rather than an ID? if this is cloned repeatedly, the ID will show up multiple times on the page, which is invalid markup